src/codegen/src/main/scala/PySparkWrapperTest.scala

```diff
@@ -261,7 +261,7 @@ class SparkTransformerWrapperTest(entryPoint: Transformer,
         case "ComputePerInstanceStatistics" => computeStatisticsString(entryPointName)
         case "IndexToValue" => indexToValueString(entryPointName)
         case "ValueIndexerModel" => valueIndexerModelString(entryPointName)
-        case "_CNTKModel" | "FastVectorAssembler" | "MultiNGram" | "ImageFeaturizer"
+        case "_CNTKModel" | "FastVectorAssembler" | "MultiNGram" | "ImageFeaturizer" | "ImageAugmenter"
            | "_ImageTransformer" | "UnrollImage" | "HashTransform" | "StopWordsRemoverTransform"
```

Comment:
OK, I don't understand why we need to update the test each time we add a new transform. Having a single update point will be a big issue once we have more developers and each will constantly run into merge conflicts.